### PR TITLE
Enabled jshint and jscs for the repository, enable travis

### PR DESCRIPTION
**Pull request by Pchelolo**
jshint and jscs checks were added for the repo, this the same config as used in RESTBase.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,7 @@
   },
   "scripts": {
     "start": "./service-runner.js",
-    "test": "mocha",
-    "coverage": "istanbul cover _mocha -- -R spec",
-    "coveralls": "cat ./coverage/lcov.info | coveralls"
+    "test": "mocha ./test/index.js"
```
Comment (d00rman) [inline package.json]: Just `mocha` is enough.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
```
Comment (d00rman) [inline .travis.yml]: I don't think there's a need to check for both node versions, we are only statically checking the code.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    - "0.12"
```
Comment (d00rman) [inline .travis.yml]: Any reason for leaving 0.12? (Not feeling too strong about this, just being curious)

**Comment by d00rman** [comment]
Needs a rebase :P

**Comment by Pchelolo** [comment]
@d00rman rebased

**Pull request merged**